serpent/game_agent.py

```diff
@@ -164,6 +191,56 @@ def handle_collect_frames_for_context(self, game_frame, **kwargs):
 
         time.sleep(interval)
 
+    def handle_record(self, game_frame, **kwargs):
+        keyboard_events = self.input_controller.capture_keys(duration=1 / self.config["fps"])
+        # TODO: mouse capture (no segfault plz)
+
+        game_frame_buffer = FrameGrabber.get_frames(
+            [0, 4, 8, 12],
```

Comment:
Might want to customize how many / which frames we are grabbing for the sequence